app/javascript/components/payment-form.js

```diff
@@ -0,0 +1,82 @@
+import { u } from 'umbrellajs';
+import { loadScript } from '../utils';
+
+const FONT_SIZE = 16;
+
+function createCardElement(element, stripe) {
+  // Create an instance of Elements.
+  const elements = stripe.elements();
+
+  const style = {};
+
+  const classes = {};
+
+  // Create an instance of the card Element.
+  const card = elements.create('card', {classes, style});
+
+  card.mount(element);
+
+  return card;
+}
+
+// Handle real-time validation errors from the card Element.
+function addListenersToCard(card, errorContainer) {
```

Comment:
@TimKJones I copied this from my other project since I did the same charge logic